content/documentation/install-uninstall.md

```diff
@@ -6,116 +6,125 @@ abstract: "Get set up to use Quire on macOS, Window, or Linux"
+Installing Quire requires using Windows PowerShell {{< q-def "command-line shell" >}} as an Administrator. Open it by right clicking on Start and selecting “Windows PowerShell (Admin)”, or search for it in the search bar and select “Run as Administrator”. If you are new to the command-line, read our tutorial [*Working in a Command-line Shell*](/learn/tutorial/#1-working-in-a-command-line-shell).
 
-2. **Node.js** and **npm** enable you to run javascript on your computer. Download and install Node.js and npm at https://nodejs.org/en/download/. Make sure you get the LTS version of Node.js, npm will be installed during the same process. The Windows installer will be downloaded, just open it and a setup wizard screen will guide you through the process. Node.js
+### Quick Install
 
-3. **PrinceXML** enables you to create a PDF version of your Quire project. You will need to download either the 32-bit installer or the 64-bit installer depending on your operating system. To install Prince you can follow the instructions at their site: https://www.princexml.com/doc-install/#windows
+If you’re eager to get started, this will install the complete Quire package but without e-book or PDF output capability. These may be added later, by following steps 3–5 in the Full Install guidelines below.
 
-4. **Pandoc** serves two purposes in Quire. You can use it to convert Word documents to Markdown, and you can also use it to create e-book files of your Quire project. You will need to download either the 32-bit installer or the 64-bit installer depending on your operating system. To install Pandoc you can either download the .msi or download the .zip file and run the .exe file in the directory. Either way works and will install Pandoc onto your system.
+1. Download and install Git for Windows: https://gitforwindows.org/
+2. Download and install the **LTS** version of Node.js: [https://nodejs.org](https://nodejs.org)
+3. In Windows PowerShell (Admin), install Windows Build Tools with: `npm install --g --production windows-build-tools`
+4. In Windows PowerShell (Admin), install Quire with: `npm install --global @thegetty/quire-cli`
+5. Confirm installation: `quire --help`
 
-5. Open **PowerShell** (that should be installed by default in your Windows computer) and run it as administrator. To do so, right click on the icon and select "run as administrator" from the context menu.
+The Quire installation process may take several minutes, during which time there will be considerable messaging output in PowerShell. The only messages of any concern are those labeled as ERROR or ERR. These likely indicate a failed installation, as would seeing "command not found" after entering `quire --help` in step 5. Search or post to our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.
```

Comment:
and would say "Command not found" ?

Reply:
I don't think I understand your query here. Is it just that I should capitalize Command?

Reply:
Ah, I reread it a few times... the phrasing "as would seeing" confused me but it does make sense. My apologies!

Reply:
Oh! How about:

> If you see these errors, or if you see "command not found" after entering `quire --help` in step 5., search our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.

Reply:
Yes, I like "if you see 'command not found'..."